.github/workflows/ci.yml

```
@@ -432,7 +432,7 @@ jobs:
         run: |
           fuel-core run --db-type in-memory --debug &
           sleep 5 &&
-          cargo run --locked --release --bin test -- --locked --release
+          cargo run --locked --release --bin test -- --locked --release --verify-ir=all
```

Nitpick: we don't use the `=` syntax for setting params anywhere else so it feels a bit odd.
```suggestion
          cargo run --locked --release --bin test -- --locked --release --verify-ir all
```